packages/math/package.json

```diff
@@ -27,7 +27,6 @@
     "test": "jest --passWithNoTests",
     "lint": "eslint \"src/**/*\" && prettier --check \"src/**/*\"",
     "lint:fix": "eslint --fix \"src/**/*\" && prettier --write \"src/**/*\"",
-    "prepare": "yarn build && yarn test",
```

Comment:
FYI this was used by lerna to publish to NPM I believe... not sure if that gets broken

Reply:
👍 Thanks for letting me know! I brought it back